plugins/field-bitmap/src/field-bitmap.ts

```diff
@@ -60,6 +73,11 @@ export class FieldBitmap extends Blockly.Field<number[][]> {
       // Set a default empty value
       this.setValue(this.getEmptyArray());
     }
+    if (config?.fieldHeight) {
+      this.pixelSize = config.fieldHeight / this.imgHeight;
```

Comment:
I'm not not a CSS expert.  Do fractional sizes work?  Does the total height come out correctly, or might rounding result in the actual total height being quite different than `fieldHeight`?  It's OK if so but should probably be documented.

Reply:
Great question! It seems to vary just a bit depending on the browser. Will document.